inc/Resizable.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
I guess that is a nit-pick, but, to me, `Resizable` indicates that an instance of the implementing class can be resized.
Instead, I would opt for something like `ResizingProvider` to clearly indicate that the provider does/can resize (images).

Reply:
Good shout, maybe `Resize`? So the pattern would be to use the verb for the capability being added. As that's the core method being added and I think it'll be similar for other interfaces too. We'd have code like:

```php
class WPProvider extends Provider implements Resize, Upload, Edit {
   // ...
}
```

I like how that reads closer to a sentence personally. Think I'll add a folder called `interfaces` and put them in there under an `AssetManagerFramework\Interfaces` namespace so it's clear what they are.